src/credentials/FabricTable.h

```diff
@@ -368,11 +383,18 @@ class DLL_EXPORT FabricTable
      */
     CHIP_ERROR AddNewFabric(FabricInfo & fabric, FabricIndex * assignedIndex);
 
+    /**
+     * Obtain a pointer to the FabricInfo. Note that its possible for fabrics to be added
+     * and removed, and for the FabricTable to be re-inited from storage. As a result
+     * clients should use FindFabric when a FabricInfo is needed and avoid holding a
+     * reference since the underlying Fabric may change.
```

Comment:
This is not viable.  Consumers hold pointers to FabricInfo* all over.

Fabric removal notifies so they can update.  Fabric addition does not invalidate existing pointers.

Reply:
reverted this approach